feat(views): make layout header responsive

hypeJunction opened the pull request:
Replaces floats with flexbox. Improves spacing between layout
heading and menu

jdalsem commented on views/default/elements/layout.css.php:
```diff
@@ -86,6 +86,29 @@
 }
+
+.elgg-layout-header {
+  border-bottom: 1px solid #EBEBEB;
```
use tabs instead of spaces

hypeJunction replied:
hm. odd, I thought they were...

hypeJunction committed:
feat(views): make layout header responsive

Replaces floats with flexbox and improves spacing between layout
heading and menu

hypeJunction commented:
@jdalsem address the tab issue
